frontend/src/pages/People/components/MemberItem.tsx

```diff
@@ -42,10 +42,23 @@ const useStyles = makeStyles((theme) => ({
 export default function MemberItem(props: MemberItemProps) {
   const member = props.member
   const memberLength = member.publicAddress.length
-  // console.log('member', member)
   const classes = useStyles()
   const { showToast } = useToast()
   const { account } = useWeb3()
+  const { provider } = useWeb3()
+  const [ENS, setENS] = React.useState('')
+
+  const userENS = async (address: string) => {
```

Comment:
OK, this is helpful for the current user profile box.

We should also show ENS data for all people cards on the People page that have ENS entries available.

Reply:
See this thread. Using subgraph to pull up the unique ENS domain (if any) matching a user address.
https://github.com/ensdomains/ens-subgraph/issues/25#issuecomment-876833128

Reply:
Pulling up ENS domain for every user is already done.